bats/core/api/ln-receive.bats

```diff
@@ -0,0 +1,380 @@
+#!/usr/bin/env bats
+
+load "../../helpers/cli.bash"
+load "../../helpers/user.bash"
+load "../../helpers/onchain.bash"
+load "../../helpers/ln.bash"
+load "../../helpers/wallet.bash"
+load "../../helpers/callback.bash"
+load "../../helpers/subscriber.bash"
+
+setup_file() {
+  lnds_init
+
+  create_user 'alice'
+  user_update_username 'alice'
+  fund_user_onchain 'alice' 'btc_wallet'
+  fund_user_onchain 'alice' 'usd_wallet'
+  add_callback 'alice'
+
+  fund_user_lightning 'alice' 'btc_wallet' '500000'
+
+  subscribe_to 'alice' my-updates-sub
+  sleep 3
```

Comment:
Why sleep 3? Can we have a loop to detect when we're done. Sleep tends to be flaky.